src/ResourceManager/ApiManagement/Commands.ApiManagement.ServiceManagement/Commands/SetAzureApiManagementLogger.cs

```diff
@@ -14,41 +14,48 @@
 
 namespace Microsoft.Azure.Commands.ApiManagement.ServiceManagement.Commands
 {
-    using Microsoft.Azure.Commands.ApiManagement.ServiceManagement.Models;
-    using Microsoft.Azure.Management.ApiManagement.SmapiModels;
+    using Management.ApiManagement.Models;
+    using Microsoft.Azure.Commands.ApiManagement.ServiceManagement.Models;    
     using System;
     using System.Collections.Generic;
     using System.Management.Automation;
 
     [Cmdlet(VerbsCommon.Set, Constants.ApiManagementLogger)]
     [OutputType(typeof(PsApiManagementLogger))]
     public class SetAzureApiManagementLogger : AzureApiManagementCmdletBase
-    {
+    {        
         [Parameter(
             ValueFromPipelineByPropertyName = true,
             Mandatory = true,
             HelpMessage = "Instance of PsApiManagementContext. This parameter is required.")]
         [ValidateNotNullOrEmpty]
         public PsApiManagementContext Context { get; set; }
 
-        [Parameter(
+        [Parameter(           
             ValueFromPipelineByPropertyName = true,
             Mandatory = true,
             HelpMessage = "Identifier of logger to update. This parameter is mandatory.")]
         [ValidateNotNullOrEmpty]
         public String LoggerId { get; set; }
 
-        [Parameter(
+        [Parameter(     
             ValueFromPipelineByPropertyName = true,
             Mandatory = false,
             HelpMessage = "EventHub Entity name. This parameter is optional.")]
         public String Name { get; set; }
 
-        [Parameter(
+        [Parameter(            
             ValueFromPipelineByPropertyName = true,
             Mandatory = false,
             HelpMessage = "EventHub Connection String with Send Policy Rights. This parameter is optional.")]
         public String ConnectionString { get; set; }
+       
+        [Parameter(            
+            ValueFromPipelineByPropertyName = true,
+            Mandatory = false,
+            HelpMessage = "Instrumentation Key of the application Insights. This parameter is optional.")]
+        [ValidateNotNullOrEmpty]
+        public String InstrumentationKey { get; set; }
```

Comment:
@solankisamir do we need to put this parameter in a separate parameter set from `ConnectionString` like we did for the `New-*` cmdlet? This will avoid allowing users to provide both in the same set